tests/integration/incremental_schema_tests/test_incremental_schema.py

```diff
@@ -87,6 +87,21 @@ def run_incremental_append_new_columns(self):
         self.list_tests_and_assert(select, exclude, expected)
         self.run_tests_and_assert(select, exclude, expected, compare_source, compare_target)
 
+    def run_incremental_append_new_columns_remove_one(self):
```

Comment:
Add to test below:
```python
    @use_profile('bigquery')
    def test__bigquery__run_incremental_append_new_columns(self):
        self.run_incremental_append_new_columns()
        self.run_incremental_append_new_columns_remove_one()
```

Reply:
Indeed I missed that 😵‍💫 
I updated it 👍